gm4_iacio_shamir/data/gm4_iacio_shamir/functions/init.mcfunction

```diff
@@ -0,0 +1,12 @@
+scoreboard objectives add gm4_ml_iacio_tri minecraft.used:minecraft.trident
+scoreboard objectives add gm4_ml_iacio_itm minecraft.custom:minecraft.drop
+scoreboard objectives add gm4_ml_iacio_prl minecraft.used:minecraft.ender_pearl
+scoreboard objectives add gm4_ml_iacio_snw minecraft.used:minecraft.snowball
+scoreboard objectives add gm4_ml_iacio_egg minecraft.used:minecraft.egg
+scoreboard objectives add gm4_ml_iacio_sp minecraft.used:minecraft.splash_potion
+scoreboard objectives add gm4_ml_iacio_lp minecraft.used:minecraft.lingering_potion
```

Comment:
`ml_` shouldn't be used as a prefix unless code is part of the metallurgy base.  This change needs to be made in multiple sections of code throughout the module.